.nycrc.json

```diff
@@ -1,4 +1,9 @@
 {
     "cache": true,
-    "reporter": "cobertura"
+    "reporter": "cobertura",
+    "exclude": [
+        "src/codec/custom/*.ts",
+        "src/codec/*.ts",
+        "test/**"
```

Comment:
default exclude is replaced, so we need to add test folder here. https://github.com/istanbuljs/schema/blob/master/default-exclude.js

Reply:
I tried to add "include": "src/**" etc. all variants of it, but it did not work(produced empty coverage). Whatever, this version works for us for now

Reply:
what about other folders like benchmarks code_samples etc.? Are they also included now?

Reply:
no they are not, since they are not visited during test run